src/SignalR/server/Core/src/HubInvocationContext.cs

```diff
@@ -2,20 +2,41 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A context passed to <see cref="IAuthorizationHandler"/>'s for custom Authorization on Hub methods.
```

Comment:
I'd be more general here. It will also be the type we use for the pipeline when/if we do that.

```suggestion
    /// Context for a Hub invocation.
```

Reply:
Yes